ass/build.gradle

```diff
@@ -23,8 +23,9 @@ dependencies {
     implementation "com.android.support:design:27.1.1"
     implementation "com.android.support.constraint:constraint-layout:1.1.3"
 
-    implementation "com.squareup.retrofit2:retrofit:2.5.0"
-    implementation "com.squareup.retrofit2:converter-moshi:2.5.0"
+    implementation "com.squareup.retrofit2:retrofit:2.7.0"
+    implementation "com.squareup.retrofit2:converter-moshi:2.7.0"
+    kapt "com.squareup.moshi:moshi-kotlin-codegen:2.7.0"
```

Comment:
please, try to check the code that you push. or at least try to build the project. Version 2.7.0 of moshi does not exist. And add also regular moshi dependency, not just a codegen

Reply:
Interesting it did not fail during sync

Reply:
I think for kapt you need to run the project and it will fetch the dependencies when it wants to use them

Reply:
fixed